browser/net/ipfs_redirect_network_delegate_helper_unittest.cc

```diff
@@ -405,4 +405,97 @@ TEST_F(IPFSRedirectNetworkDelegateHelperTest, PrivateProfile) {
   EXPECT_TRUE(brave_request_info->new_url_spec.empty());
 }
 
```

Comment:
maybe also add a test if ipfs is disabled, the header is NOT added and HTTPS does NOT get reduced to HTTP

Reply:
Reduction works only for ipns.localost, so there is the test that other urls like brave.com doesn't get reduced - IpfsRedirectNetworkDelegateHelperBrowserTest::HTTPSSchemeDowngrade_CommonUrl